src/components/users/Users.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState, useEffect} from 'react'
+import { getUsers } from "../../services/userApi"
+import { cols } from "./userColumns"
+import EnhancedTable from "../DndTable/Table"
+
+function Users() {
+    const [data, setData] = useState([]);
+
+    useEffect(() => {
+    getUsers().then(res => setData(res.data.data.results));
```

Comment:
please add catch block to handle errors.

Reply:
done